config/sc_mpi.m4

```diff
@@ -23,6 +23,7 @@ dnl If MPI is enabled, set AC_DEFINE and AC_CONDITIONAL for PREFIX_ENABLE_MPI.
 dnl If MPI I/O is not disabled, set these for PREFIX_ENABLE_MPIIO.
 dnl If MPI_Init_thread is not disabled, set these for PREFIX_ENABLE_MPITHREAD.
 dnl If MPI shared nodes are supported, set these for PREFIX_ENABLE_MPISHARED.
+dnl If MPI shared memory is continious, set these for PREFIX_ENABLE_MPISOCKET.
```

Comment:
Please spell-check, also do you mean continuous or contiguous, and how does it relate to TYPE?

I was under the impression that the setting of contiguous shared memory is an independent quality from TYPE_SHARED and TYPE_SOCKET.

Reply:
Updated the description.